cmd/influxd/launcher/cmd.go

```diff
@@ -237,6 +240,12 @@ func (o *InfluxdOpts) BindCliOpts() []cli.Opt {
 			Default: o.BoltPath,
 			Desc:    "path to boltdb database",
 		},
+		{
+			DestP:   &o.SqLitePath,
+			Flag:    "sqlite-path",
+			Default: o.SqLitePath,
```

Comment:
I think we can remove this `Default` (and simplify the `Desc` below). The default sqlite path and the default bolt path are in the same directory, so if we remove the default the put-in-same-directory logic should kick in and get us back to the same result.

Reply:
Great idea...I was struggling a bit to come up with a succinct description and didn't think of just getting rid of the default. That made the description easier to write and made the logic a lot cleaner as well.